L72-core-google_default_credentials-extension.md

```diff
@@ -0,0 +1,62 @@
+Allow Call Credentials to be Specified in `grpc_google_default_credentials_create`
+----
+* Author(s): rbellevi
+* Approver: markdroth
+* Status: Draft
+* Implemented in: Core
+* Last updated: July 6th, 2020
+* Discussion at: (TODO)
+
+## Abstract
+
+This document proposes that `grpc_google_default_credentials_create` be
+amended to allow the user to specify their desired call credential, while
+also adding a `void* reserved` argument to lessen the burden of any such
+changes in the future.
+
+
+## Background
+
+The so-called "Google default credentials" created by the
+`grpc_google_default_credentials_create` function in Core enables connection to
+GCP via a combination of ALTS and SSL credentials, along with a special oauth2
```

Comment:
It doesn't enable connection to GCP; it enables connection to OnePlatform services.

Reply:
Switching to "Google services" since OnePlatform doesn't seem to be a public name.